### ISSUE TITLE
Convert CI to GitHub Actions

### ISSUE THREAD

**Issue by alexellis** [inlets/inletsctl#81]
## Task 

Convert CI to GitHub Actions

## Context

Travis is removing its free tier for OSS projects, so this should be built with GitHub Actions instead, which is not going to add additional cost.

I would expect this to be one of the easier projects to convert given that Docker isn't used in the build.

See also:

https://github.com/mvdan/github-actions-golang
https://github.com/actions/create-release

https://github.com/alexellis/arkade/issues/263
https://github.com/alexellis/k3sup/issues/279

**Comment by isabellaliu77** [comment]
Can I try this?

**Comment by alexellis** [comment]
Yes please use the approach from k3sup and read the contribution guide before starting. Each commit needs to be signed off (not GPG)

**Comment by isabellaliu77** [comment]
Ok, thanks alex
